caffe2/python/operator_test/conv_test.py

```diff
@@ -289,9 +346,10 @@ def test_3d_convolution_nchw(self, input_channels, output_channels,
            dilation=st.integers(1, 2),
            pad=st.integers(0, 2),
            use_bias=st.booleans(),
-           **hu.gcs)
-    def test_3d_convolution_cudnn_nchw(self, batch_size, stride, size, kernel,
-                                       dilation, pad, use_bias, gc, dc):
+           **hu.gcs_gpu_only)
```

Comment:
I think now this test should work for both GPU & CPU can you change:
`**hu.gcs_gpu_only` -> `**hu.gcs`